docs/wsgi.rst

```diff
@@ -210,6 +210,92 @@ section will pull in the correct dependencies:
    wsgi = ${buildout:directory}/etc/bjoern.ini
 
 
+werkzeug
+~~~~~~~~
+`werkzeug <https://palletsprojects.com/p/werkzeug/>`_ is a WSGI library that
+contains not just a WSGI server, but also a powerful debugger. It can
+easily integrate wth Zope using a shim package called `dataflake.wsgi.werkzeug 
+<https://dataflakewsgiwerkzeug.readthedocs.io/>`_. See the `Using this package`
+section for how to integrate `werkzeug` using Zope's own ``runwsgi`` script and
+how to create a suitable WSGI configuration.
+
+If you use ``plone.recipe.zope2instance``, the following section will pull in
+the correct dependencies, after you have created a WSGI configuration file:
+
+.. code-block:: ini
+
+   [zopeinstance]
+   recipe = plone.recipe.zope2instance
+   eggs =
+       dataflake.wsgi.werkzeug
+   zodb-temporary-storage = off
+   user = admin:password
+   http-address = 8080
+   wsgi = ${buildout:directory}/etc/werkzeug.ini
+
+
+Debugging Zope applications under WSGI
+--------------------------------------
+You can debug a WSGI-based Zope application the same way you have debugged
+ZServer-based installations in the past. In addition, you can now take
+advantage of WSGI middleware or debugging facilities built into the chosen
+WSGI server.
+
+When developing your application or debugging, which is the moment you want to
+use debugging tools, you can start your Zope instance in `exceptions debug
+mode`. This will disable all registered exception views including
+``standard_error_message`` so that exceptions are not masked or hidden.
+
+This is how you run Zope in exceptions debug mode using the built-in
+``runwsgi`` script:
+
+.. code-block:: console
+
+   $ bin/runwsgi -ev etc/zope.ini
```

Comment:
Is the `-v` actually needed? From reading the text I was thinking the new option is called `-ev`. Maybe `--debug-exceptions` could be used here for clarity.

Reply:
I added the `-v` purely out of habit. During development it's always `bin/runwsgi -dv etc/zope.ini`, and while I know what `-d` does, I don't even know if `-v` makes any difference. I dropped it in the document.

Now I can use `bin/runwsgi -dev etc/zope.ini`, which is kind of cool ;-)